frontend/public/useragreement-sv.md

```diff
@@ -1 +1,17 @@
 ### Användaravtal
+
+“IT” refer till Teknologsektionen Informationsteknik, Chalmers Studentkår med org. Nummer 857209-9524.
+
+Detta avtal rör datahantering i IT:s kontotjänst Gamma. 
+
+IT kommer att samla in och hantera namn, Chalmers-id, smeknamn, antagningsår på Informationsteknik, e-mail, mobiltelefonnummer, hemsidor som du lägger in, vilka organ som du är med i på sektionen, vilket språk du föredrar samt bild kopplad till dig.
+
+Datan hanteras i syfte att autentisera dig mot IT:s och tredjeparts-tjänster. Datan används för att skapa en profil som kan användas på IT:s tjänster samt även de tredjepartstjänster du godkänner. Datan används även ifall vi behöver kontakta dig vid ärende av särskilt intresse för dig. Datan hanteras även i statistiska syften. 
+
+Datan sparas tills du väljer att ta bort din profil eller att IT sektionen väljer att radera din data vilket i så fall sker efter 30 dagars notis. 
```

Comment:
..du kommer dock att meddelas om detta 30 dagar innan.

Using "notis" in swedish feels wrong..